Modules with "~/..." missing in Typescript

When building a project with Typescript I get the following errors:

```
node_modules/qusly-core/build/strategies/ftp.d.ts:5:81 - error TS2307: Cannot find module '~/interfaces' or its corresponding type declarations.

5 import { IFile, IFtpConfig, ITransferOptions, IFtpOptions, ITransferInfo } from '~/interfaces';
                                                                                  ~~~~~~~~~~~~~~

node_modules/qusly-core/build/strategies/sftp.d.ts:8:82 - error TS2307: Cannot find module '~/interfaces' or its corresponding type declarations.

8 import { IFile, ITransferOptions, IFtpConfig, ISFtpOptions, ITransferInfo } from '~/interfaces';
                                                                                   ~~~~~~~~~~~~~~

node_modules/qusly-core/build/strategies/strategy.d.ts:4:83 - error TS2307: Cannot find module '~/interfaces' or its corresponding type declarations.

4 import { IFile, ITransferOptions, ITransferProgressListener, ITransferInfo } from '~/interfaces';
                                                                                    ~~~~~~~~~~~~~~

node_modules/qusly-core/build/utils/function.d.ts:1:31 - error TS2307: Cannot find module '~/interfaces' or its corresponding type declarations.

1 import { ITaskResponse } from '~/interfaces';
```

I noticed that the `~` is mapped in the `qusly-core` `tsconfig.json` and it apparently works when running `tsc` on the `qusly-core` project to generate the npm package. However, the type definitions in the npm package still contains imports from `~/interfaces`.

I guess that simply replacing `~` with relative imports (most of the imports are relative imports anyways) would solve the issue?

--- a/src/strategies/ftp.ts
+++ b/src/strategies/ftp.ts
@@ -8,8 +8,8 @@ import {
   ITransferOptions,
   IFtpOptions,
   ITransferInfo,
-} from '~/interfaces';
-import { FtpUtils } from '~/utils/ftp';
+} from '../interfaces';
+import { FtpUtils } from '../utils/ftp';
 
 export declare interface FtpStrategy {
   config: IFtpConfig;

--- a/src/strategies/sftp.ts
+++ b/src/strategies/sftp.ts
@@ -12,8 +12,8 @@ import {
   IFtpConfig,
   ISFtpOptions,
   ITransferInfo,
-} from '~/interfaces';
-import { FtpUtils } from '~/utils/ftp';
+} from '../interfaces';
+import { FtpUtils } from '../utils/ftp';
 
 export declare interface SftpStrategy {
   config: IFtpConfig;

--- a/src/strategies/strategy.ts
+++ b/src/strategies/strategy.ts
@@ -6,8 +6,8 @@ import {
   ITransferOptions,
   ITransferProgressListener,
   ITransferInfo,
-} from '~/interfaces';
-import { Transfer } from '~/transfer';
+} from '../interfaces';
+import { Transfer } from '../transfer';
 
 export declare interface Strategy {
   on(event: 'connect', listener: () => void): this;

--- a/src/utils/ftp.ts
+++ b/src/utils/ftp.ts
@@ -1,6 +1,6 @@
 import { FileInfo, FileType } from 'basic-ftp';
 
-import { IFile, IFileType } from '~/interfaces';
+import { IFile, IFileType } from '../interfaces';
 
 const MSDOS_DATE_REGEX = /[0-9][0-9]-[0-9][0-9]-[0-9][0-9]\s[0-9][0-9]\:[0-9][0-9][PA]M/gi;
 

--- a/src/utils/function.ts
+++ b/src/utils/function.ts
@@ -1,4 +1,4 @@
-import { ITaskResponse } from '~/interfaces';
+import { ITaskResponse } from '../interfaces';
 
 export const execFunction = async (
   f: Function,

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -16,11 +16,7 @@
     "sourceMap": true,
 
     "rootDir": "./src",
-    "baseUrl": "./",
-
-    "paths": {
-      "~/*": ["./src/*"]
-    }
+    "baseUrl": "./"
   },
   "include": ["./src"],
   "exclude": ["node_modules", "build"]

